docs/source/getting_started/getting_started.md

```diff
@@ -18,32 +18,44 @@ We have two authentication tokens, one to connect to the back office and one to
 To authenticate, we need to set the `authenticationToken` header to one of them, depending on what we need to do.
 As our API expects all request bodies to be JSON encoded, we also need to set the `Content-Type` header to `application/json`.
 
+In the follwoing examples we are going to use three environment variables: `$URL`, `$BACKOFFICE_TOKEN`, and `$FRONTOFFICE_TOKEN`.
+To try the examples you need first to set them to the values for your system:
+
+```bash
+export URL="<url>"
+export BACKOFFICE_TOKEN="<backoffice_token>"
+export FRONTOFFICE_TOKEN="<frontoffice_token>"
+```
+
 # Ingest
 
 We can use the back office endpoint [`/documents`](https:/docs.xayn.com/back_office.html#operation/createDocuments) to ingest documents.
 
 We will ingest a document that represents this article: [https://xayn.com/blog/the-initial-challenge](https://xayn.com/blog/the-initial-challenge).
 
 ```bash
-curl -X POST https://<url>/documents
-    -H "authorizationToken: <back_office_token>"
-    -H "Content-Type: application/json"
+curl -X POST "$URL/documents" \
+    -H "authorizationToken: $BACKOFFICE_TOKEN" \
+    -H "Content-Type: application/json" \
     -d '{
+
         "documents": [
             {
                 "id": "xayn_cd5604c",
-                "snippet": "The voices that are demanding better privacy protection and ownership of our own data are increasingly louder, there's a backlash towards these practices. At Xayn, our mission is to provide personalisation without user data leaving the device, maintaining absolute privacy. We use semantic similarity and centers of interest to understand user preferences and present better matching articles. With our model Xaynia, we offer semantic similarity and search with minimal energy consumption and at a low price, making it highly energy-efficient compared to other transformer models.",
```

Comment:
Transformed `there's` to `there is` since we cannot escape in a single-quoted string.